### DISCUSSION TITLE
Fixed component

### DISCUSSION THREAD

**Discussion by laurent512** [jscottsmith/react-scroll-parallax#232] [General]
Hello, 

I am willing to code a landing with with multiple sections , each section takes 100vh. 
The first section is just a picture, that should not move when scroll is occuring. 
That would give an effect of the second section coming on top of it.  

How to do so , is it like Yspeed was -inf ?

**Comment by jscottsmith** [comment]
Hi @laurent512 

If I understand your intention correctly, you do not need the `react-scroll-parallax` at all. This can be achieved through CSS like so:

https://codesandbox.io/s/fixed-background-hero-example-9w75gq
